atm.rb

```diff
@@ -0,0 +1,146 @@
+require "pry"
+class Menu
+  # puts "Enter username, please."
+  # username=gets.chomp
+  # puts "Enter password, please."
+  # password=gets.chomp
+  @@debitCard= []
+  @@creditCard=[]
+  @@totaldebit=0
+  @@totalcredit=0
+  $eachTransaction={}
+
+
+  def initialize
+    @@activity=activity
```

Comment:
Where do these local `activity` and `debitCredit` variables come from? Are they supposed to be populated by arguments to `initialize`?